rednotebook/gui/clouds.py

```diff
@@ -178,20 +179,20 @@ def _get_tags_for_cloud(self, tag_count_dict, ignores):
         tags_and_frequencies = [
             (tag, freq)
             for (tag, freq) in tag_count_dict
-            if not any(pattern.match(tag) for pattern in ignores)
+            if all(not pattern.match(tag) for pattern in ignores)
```

Comment:
I like the `not any()` version better (also below) since it almost reads like English text.